core/tests/classes/libs/input/InputTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+/**
+ * KumbiaPHP web & app Framework
+ *
+ * LICENSE
+ *
+ * This source file is subject to the new BSD license that is bundled
+ * with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://wiki.kumbiaphp.com/Licencia
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@kumbiaphp.com so we can send you a copy immediately.
+ *
+ * @category   Kumbia
+ * @package    Session
+ * @copyright  Copyright (c) 2005 - 2017 Kumbia Team (http://www.kumbiaphp.com)
+ * @license    http://wiki.kumbiaphp.com/Licencia     New BSD License
+ */
+
+/**
+ * @category Test
+ */
+class InputTest extends PHPUnit_Framework_TestCase
```

Comment:
__[PHP_CodeSniffer]__

Each class must be in a namespace of at least one level (a top-level vendor name)


<!-- Comment by [SideCI](https://sideci.com) -->